Service/RequestService.php

```diff
@@ -547,7 +548,7 @@ public function requestHandler(array $data, array $configuration): Response
 
         $this->handleMetadataSelf($result, $metadataSelf);
 
-        $result = $this->shouldWeUnsetEmbedded($result, $this->data['headers']['accept'] ?? null, $isCollection ?? false);
+//        $result = $this->shouldWeUnsetEmbedded($result, $this->data['headers']['accept'] ?? null, $isCollection ?? false);
```

Comment:
Dit willen we volgensmij alleen voor kiss doen en niet op master.
Dit zorgt dat we altijd embedded tonen namelijk.
Waar, al dus Barry, we voor roxit deze functie gebruiken en alleen voor accept header json+ld embedded tonen